extensions/pyRevitTools.extension/checks/radar_check.py

```diff
@@ -0,0 +1,484 @@
+def get_project_base_and_survey_pts(document=doc):
+    base_point = DB.BasePoint.GetProjectBasePoint(document).Position
+    base_point_coordinates = (
+                                base_point.X,
+                                base_point.Y,
+                                base_point.Z)
+    survey_point = DB.BasePoint.GetSurveyPoint(document).Position
+    survey_point_coordinates = (
+                                survey_point.X,
+                                survey_point.Y,
+                                survey_point.Z)
+    return base_point_coordinates, survey_point_coordinates
+
+
+# _______________________________________________________________Get Model Units
+def get_model_units_type(document=doc):
+    unitsystem = document.DisplayUnitSystem
+    return unitsystem
+
+
+# _________________________________________________ Get Design Options & Objects
+def get_design_options_elements(document=doc):
+    design_option_elements = []
+    design_option_sets = []
+    design_options = (DB.FilteredElementCollector(document).
+                        OfClass(DB.DesignOption).
+                        ToElements())
+
+    for do in design_options:
+        option_set_param = DB.BuiltInParameter.OPTION_SET_ID
+        option_set_id = document.get_Parameter(option_set_param).AsElementId()
+        design_option_sets.append(option_set_id)
+        design_option_filter = DB.ElementDesignOptionFilter(do.Id)
+        x = (DB.FilteredElementCollector(document).
+                WherePasses(design_option_filter).
+                ToElements())
+        design_option_elements.append(x)
+    return design_options, design_option_elements, design_option_sets
+
+
+# ______________________________________________________________________________
+# ________________________________________________________________ MAIN FUNCTION
+# ______________________________________________________________________________
+def check_model_extents(document=doc):
+    unit_system = get_model_units_type(document)
+    # _______________________________________________________________HTML Styles
+    output = script.get_output()
+    output.add_style('cover {color:black; font-size:24pt; font-weight:bold;}')
+    output.add_style('header {color:black; font-size:15pt;}')
+    divider = "_"*100
+    test_score = 0
+    output.print_html(
+     ('<cover>__________:satellite_antenna:__10-Mile Radar___________</cover>'))
+    print(divider)
+    print("")
+    output.print_md('# Checking model placement and coordinates')
+    print(divider)
+    # _____________________________Check the distances of base and survey points
+    baspt, survpt = get_project_base_and_survey_pts(document)
+    basptdistance = abs(calculate_distance(baspt, INTERNAL_ORIGIN))
+    surveydistance = abs(calculate_distance(survpt, INTERNAL_ORIGIN))
+
+    if basptdistance > EXTENT_DISTANCE:
+        output.print_md(BAD_STRG + 
+        'Base Point is more than ' + CRITERIA_STRG)
```

Comment:
This is valid for all the other lines below: use the `str.format()` method instead of string concatenation